### PR TITLE
Rename ocid to crio

### PR DESCRIPTION
**Pull request by rhatdan**
We want to elminate the name ocid from use.  Move fully to crio.

Also cric is being renamed to crioctl for the time being.

### CODE REVIEW

**Comment by mrunalp** [comment]
Commit needs to be signed and gofmt server/config.go

**Comment by rhatdan** [comment]
Fixed.

@mrunalp PTAL

File: cmd/criod/main.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
-const ociConfigPath = "/etc/ocid/ocid.conf"
+const ociConfigPath = "/etc/criod/criod.conf"
```
Comment (mrunalp) [inline cmd/criod/main.go]: ociConfigPath -> criodConfigPath

**Comment by mrunalp** [comment]
I am having some second thoughts about the name `criod`. Should we just keep it simple and call it `crio`. The client is a test client and its name doesn't matter as much. It could be renamed to `crioctl`. 
WDYT?

**Comment by mheon** [comment]
Personally would be in favor of `crio` over `criod` - there are plenty of projects that use their project name for the main executable (nginx, for example), and `crio` is easier to say and sounds a lot better than `criod` to me.

**Comment by runcom** [comment]
+1 to `crio` and `crioctl`

**Comment by mrunalp** [comment]
Also, one of the tests failed:
```
not ok 28 Ensure correct CNI plugin namespace/name/container-id arguments
```

**Comment by jawnsy** [comment]
Is `crioctl` meant only for CRI-O or is it intended as a general tool for CRI implementations? I thought the `crictl` name suggested the latter, especially given that it's in its own repository? https://github.com/kubernetes-incubator/cri-tools

Or will there be *both* `crioctl` and `crictl`?

**Comment by sameo** [comment]
@jawnsy 
> Or will there be both crioctl and crictl?

The implicit plan is to eventually drop crioctl in favor or crictl. We need to port all our tests to it for example.

**Comment by rhatdan** [comment]
@mrunalp Passes tests, I ran through this once again, and now everything is crio and crioctl.

**Comment by mrunalp** [comment]
Nice! I think we will run the k8s e2e on this once.

File: docs/crio.conf.5.md [hunk 2]
```diff
@@ -64,10 +64,10 @@ The `ocid` table supports the following options:
...
 
 **seccomp_profile**=""
-  Path to the seccomp json profile to be used as the runtime's default (default: "/etc/ocid/seccomp.json")
+  Path to the seccomp json profile to be used as the runtime's default (default: "/etc/crio/seccomp.json")
 
 **apparmor_profile**=""
-  Name of the apparmor profile to be used as the runtime's default (default: "ocid-default")
+  Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
 
 ## OCID.IMAGE TABLE
```
Comment (TomSweeneyRedHat) [inline docs/crio.conf.5.md]: Does OCID.IMAGE TABLE need to change also?

File: cmd/crio/main.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
-const ociConfigPath = "/etc/ocid/ocid.conf"
+const ociConfigPath = "/etc/crio/crio.conf"
```
Comment (TomSweeneyRedHat) [inline cmd/crio/main.go]: maybe crioConfigPath?

**Comment by TomSweeneyRedHat** [comment]
LGTM, couple of possible nits.  Is there any customer facing documentation that we need to touch up that's outside of these changes?

**Comment by mrunalp** [comment]
All tests passed with kubernetes e2e node conformance tests. We can address the minor comments in a separate PR.

**Pull request merged**